cypress/integration/obituary_spec.js

```diff
@@ -1,9 +1,9 @@
 describe('Obituary', () => {
     beforeEach(() => {
-      cy.intercept("GET", "https://elegy-backend.herokuapp.com/api/v1/users/2", {
+      cy.intercept("GET", "https://elegy-backend.herokuapp.com/api/v1/users/4", {
           ok: true,
           status: 200, 
```

Comment:
I wonder why this `status` works but the other needs to be `statusCode`?

Reply:
Maybe the default is 200?